WordPress/gutenberg
# Gallery: Refactor "Settings" panel of Gallery block to use `ToolsPanel` instead of `PanelBody`

Fixes: #67893

## What?
This PR introduces the usage of `ToolsPanel` instead of `PanelBody` inside the `Settings` panel of `Gallery` Block.

## Why?
This PR is a part of #67813 which standardizes the usage of `ToolsPanel` inside `Block Inspector Settings`

## How?
`PanelBody` is replaced with the corresponding `ToolsPanel` component.

## Testing Instructions
1. Insert a Gallery Block.
2. Notice the presence of `ToolsPanel` component.
3. Try to reset the updated values to test the reset functions.

## Screencast

https://github.com/user-attachments/assets/86c95541-9ca7-45e4-80e1-7ace331b0e71
